lib/collections.js

```diff
@@ -32,6 +38,26 @@ var uniqueWords = [];
 
 /* your code starts here */
 
+for (var i = 0; i < normalizedWords.length; i++) {
+  //if (sorted_words[i] !== sorted_words[i + 1]) {
+  uniqueWordsAsKeys[normalizedWords[i]] = true;
+}
+
+//console.log(uniqueWordsAsKeys);
+
+var i = 0;
+for (var word in uniqueWordsAsKeys) {
+  uniqueWords[i] = word;
+  i++;
```

Comment:
This will work, but there's already an easy way to add something to the end of an array - `.push()`.